src/pages/Checkout.tsx

```diff
@@ -27,6 +29,12 @@ export const Checkout = () => {
   const theme = useTheme();
   const isDesktop = useResponsive('up', 'md');
   const { checkout, account } = useAppState();
+  const { latestQueryParams } = useAccommodationsAndOffers();
+  const query = useMemo(
+    () => createSearchParams(JSON.stringify(latestQueryParams)),
```

Comment:
`latestQueryParams` can be `undefined`. `JSON.stringify(latestQueryParams)` will produce a string `undefined`.
Is it OK to pass "undefined" to the `createSearchParams` utility?